pkg/config/autodiscover/csv_info_test.go

```diff
@@ -57,3 +58,106 @@ func TestCSVGetAnnotationValue(t *testing.T) {
 	assert.Equal(t, []string{"OPERATOR_STATUS", "ANOTHER_TEST"}, val)
 	assert.Nil(t, err)
 }
+
+func TestAnnotationUnmarshalError(t *testing.T) {
+	testCases := []struct {
+		expectedString string
+	}{
+		{
+			expectedString: "error (this is an error) attempting to unmarshal value of annotation 'testKey' on CSV 'testnamespace/testname'",
+		},
+	}
+
+	for _, tc := range testCases {
+		csvr := CSVResource{
+			Metadata: struct {
```

Comment:
This is not important, but I'm curious: is it necessary to re-declare the whole struct types for the initialization? I'd just do this:
```
csvr := CSVResource{}
csvr.Name = "testname"
csvr.Namespace = "testnamespace"
```

Reply:
Yeah that was an interesting auto-fill by `gofmt`.  Not sure why it brought all of that in.  I can double check if removing it causes any issues.

Reply:
Nope, it doesn't like it when I remove it.